applications/tari_validator_node/src/consensus_constants.rs

```diff
@@ -22,12 +22,14 @@
 
 pub struct ConsensusConstants {
     pub base_layer_confirmations: u64,
+    pub validator_node_timeout: u64,
 }
 
 impl ConsensusConstants {
-    pub const fn devnet() -> Self {
+    pub const fn devnet(validator_node_timeout: u64) -> Self {
```

Comment:
I wouldnt mix the DAN constants and the base layer constants. This value may change as the base layer progresses anyhow.
I think get the base layer constants as needed in the vn registration code for now

Reply:
Refactored this